src/main/java/com/ucl/imaginethisserver/Controller/UserController.java

```diff
@@ -0,0 +1,43 @@
+package com.ucl.imaginethisserver.Controller;
+
+import com.ucl.imaginethisserver.Model.User;
+import com.ucl.imaginethisserver.Service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+@RestController
+@CrossOrigin("http://localhost:3000")
```

Comment:
Also, I'd configure CORS globally in one place, like they mention in [here](https://spring.io/blog/2015/06/08/cors-support-in-spring-framework). It would improve the code quality in other places as well.